lib/global_uid.rb

```diff
@@ -9,6 +10,8 @@ module GlobalUid
   class NoServersAvailableException < StandardError ; end
   class ConnectionTimeoutException < StandardError ; end
   class TimeoutException < StandardError ; end
+  class InvalidIncrementException < StandardError ; end
+  class InvalidIncrementPreventionException < StandardError ; end
```

Comment:
These definitely need a note at least about the difference. (That Prevention implies nothing is actually allocated). I do question the need for differentiation here, do we actually care if we lost a few ids :D

Reply:
You're the second person to mention it, so its definitely not obvious. Since that's the case, I'll just remove it and keep to the one exception class for simplicity.